ckanext/harvest/queue.py

```diff
@@ -203,7 +203,13 @@ def send(self, body, **kw):
         value = json.dumps(body)
         # remove if already there
         if self.routing_key == get_gather_routing_key():
-            self.redis.lrem(self.routing_key, 0, value)
+            # it appears that both types of call are possible within the redis library depending on which version used
+            # for now support both versions
+            # https://github.com/andymccurdy/redis-py#client-classes-redis-and-strictredis
+            try:
+                self.redis.lrem(self.routing_key, 0, value)
+            except:
```

Comment:
Can you capture the specific exception that the wrong version causes, rather than a blanket except? Cheers!